packages/devtools_app/lib/src/inspector/inspector_controller.dart

```diff
@@ -620,6 +620,26 @@ class InspectorController extends DisposableController
     try {
       final RemoteDiagnosticsNode newSelection = await pendingSelectionFuture;
       if (group.disposed) return;
+      if (newSelection == null && firstFrame) {
```

Comment:
does this code play well with the selection from query parameters above? i.e. can a widget still be selected from a query parameter on load or will the root widget be selected instead.

Reply:
yep. it only triggers if the selection is null. If query parameters are a busted selection it will mean the root is selected.